MAINTAINERS.md

```diff
@@ -77,8 +87,66 @@ Make frequent project releases to the community.
 
 ### Promote Other Maintainers
 
-Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers.
+Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers. See [Becoming a Maintainer](#becoming-a-maintainer) for more information.
 
 ### Describe the Repo
 
 Make sure the repo has a well-written, accurate, and complete description. See [opensearch-project/.github#38](https://github.com/opensearch-project/.github/issues/38) for some helpful tips to describe your repo.
+
+## Becoming a Maintainer
+
+You can become a maintainer by actively [contributing](CONTRIBUTING.md) to any project, and being nominated by an existing maintainer.
+
+### Nomination
+
+Any current maintainer starts a private e-mail thread (until we have a better mechanism, e-mails can usually be found via MAINTAINERS.md + DCO) with all other maintainers on that repository to discuss nomination using the template below. In order to be approved, at least 3 positive maintainer votes are necessary, and no vetoes. In rare cases when there are fewer than 3 maintainers, the positive votes from all maintainers are required. Any disagreements can be escalated to the repo admin.
+
+The nomination should clearly identify the person with their real name and a link to their GitHub profile, and the rationale for the nomination, with concrete example contributions.
+
+### Interest
+
+Upon receiving at least 3 positive maintainer votes, and no vetoes, from other maintainers within a maximum period of 2 weeks, the nominating maintainer asks a potential nominee whether they might be interested in becoming a maintainer on the repository, which typically happens via e-mail obtained via DCO, but may also be a comment to a pull request, following a substantial contribution.
```

Comment:
s/3/three
s/2/two

Remove "typically" and make it more concrete: `, which happens via private e-mail message`, and remove the `, but may also be a ...`?

Reply:
Why would it be a problem to ask someone to join as a maintainer in public?

Reply:
While the primary issue is the public *discussion* of the merits of an individual, not the public invitation itself, I would offer one perspective on why a public invitation can be harmful. It creates a situation where now everyone knows that they've been invited, and there will (could?) be pressure from their employer to accept an invitation that they don't have the time or inclination to accept. A private invitation allows the individual to make the decision in their own time, in their own way, without external pressure or judgement. Granted, this may be a non-issue for many/most people, but I feel that it's the more sensitive approach.

Reply:
> Why would it be a problem to ask someone to join as a maintainer in public? I do this all the time on my projects and people sometimes say they just don't have bandwidth for it.

I think this is in reference to me asking to remove the `, but...` clause right?

Multiple reasons:
  * Inviting a strong community contributor to become a possible new maintainer should not be seen by the community as an ad-hoc comment on a PR, but rather a well thought out discuss/VOTE, privately, among the existing committers (yes, in response to a strong code or community contribution, or earned merit from strong community engagement, but nevertheless a private VOTE among the existing maintainers).  It should be a new email, announcing the successful (private) vote outcome.
  * The subsequent discussion about whether the proposed new maintainer agrees that they would like to join the project or not, should also remain private.  There are many complex personal reasons about why an individual may or may not want to join, and it is their decision, not ours, whether that is a public discussion.  I have seen so many interesting (private, legitimately) reasons over the years.
  * Can you point to "my projects" so we can make an informed comparison?  I agree for smaller informal projects, such an informal process makes sense.  But this is not that.

Reply:
Yes, these are "smaller" or "informal", indeed. But maybe we would be better served to treat our modest beginnings as such, as well and not make too big of a deal of this 🤔 

https://github.com/slack-ruby/slack-ruby-client/pull/387#issuecomment-988398343
https://github.com/alexa-js/alexa-app/pull/310#pullrequestreview-92028980
https://github.com/mongoid/mongoid-slug/pull/252#issuecomment-413296153